sdks/python/apache_beam/examples/wordcount_it_test.py

```diff
@@ -31,7 +31,7 @@ class WordCountIT(unittest.TestCase):
   @attr('IT')
   def test_wordcount_it(self):
     # Set extra options to the pipeline for test purpose
-    extra_opts = {'on_success_matcher': PipelineStateMatcher()}
```

Comment:
I was referring to [all_of and any_of](https://pypi.python.org/pypi/PyHamcrest) part of PyHamcrest